config/grafana/watcher.json

```diff
@@ -21,17 +21,254 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": 41,
   "links": [],
   "panels": [
+    {
+      "datasource": "VictoriaMetrics",
```

Comment:
And you have set the treshold for the percentage count to 80, so below 80% its green and above its red. I would do it the other way around!

So below 80& it should be red, otherwise green. Please have a look at the screenshot how to do it.

Before:
![Screenshot 2024-04-09 at 10 50 28](https://github.com/kyma-project/lifecycle-manager/assets/48282931/486c7842-4203-4ce1-8c25-8528cc87fd98)

After:
![Screenshot 2024-04-09 at 10 50 20](https://github.com/kyma-project/lifecycle-manager/assets/48282931/76494f72-59b0-4c50-8502-c5d899a7a018)


How to:
![Screenshot 2024-04-09 at 10 48 54](https://github.com/kyma-project/lifecycle-manager/assets/48282931/d7124376-ffe0-4dcd-be98-a5e8f839b3fc)